### ISSUE TITLE
[BUG] UniGetUI displays `gaming` repo as `ging`

### ISSUE THREAD

**Issue by Daviteusz** [marticliment/UniGetUI#2650]
### Please confirm these before moving forward

- [X] I have searched for my issue and have not found a work-in-progress/duplicate/resolved issue.
- [X] I have tested that this issue has not been fixed in the latest [(beta or stable) release](https://github.com/marticliment/WingetUI/releases/).
- [X] I have checked the [FAQ](https://github.com/marticliment/WingetUI#frequently-asked-questions) section for solutions.
- [X] This issue is about a bug (if it is not, please use the [correct template](https://github.com/marticliment/WingetUI/issues/new/choose)).

### UniGetUI Version

3.1.1

### Windows version, edition, and architecture

Windows 11 Pro 26100.1457 x64

### Describe your issue

Read screenshot image in this bug report

### Steps to reproduce the issue

Read screenshot image in this bug report

### UniGetUI Log

```text
[21.08.2024 18:28:34]    __  __      _ ______     __  __  ______
                        / / / /___  (_) ____/__  / /_/ / / /  _/
                       / / / / __ \/ / / __/ _ \/ __/ / / // /
                      / /_/ / / / / / /_/ /  __/ /_/ /_/ // /
                      \____/_/ /_/_/\____/\___/\__/\____/___/
                          Welcome to UniGetUI Version 3.1.1
[21.08.2024 18:28:34]
[21.08.2024 18:28:34] Version Code:  3,11
[21.08.2024 18:28:34] Encoding Code Page set to 852
[21.08.2024 18:28:34] Using bundled GSudo at C:\Program Files\UniGetUI\Assets\Utilities\gsudo.exe
[21.08.2024 18:28:34] Loaded language locale: pl
[21.08.2024 18:28:34] Randomly-generated background API auth token for the current session: 0yc3d8zus9f9k6ks7bvkujolbbxkmq29wdp6qkh9axgzueb05imjd714pmzo24ot
[21.08.2024 18:28:35] Api running on http://localhost:7058
[21.08.2024 18:28:35] Lang files were updated successfully from GitHub
[21.08.2024 18:28:35] Downloaded new icons and screenshots successfully!
[21.08.2024 18:28:35] Command npm was not found on the system
[21.08.2024 18:28:35] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                      █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                      █ Name: Npm
                      █ Enabled: True
                      █ Found: False
                      █ THE MANAGER WAS NOT FOUND. PERHAPS IT IS NOT 
                      █ INSTALLED OR IT HAS BEEN MISCONFIGURED 
                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[21.08.2024 18:28:35] Command dotnet.exe was not found on the system
[21.08.2024 18:28:35] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                      █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                      █ Name: .NET Tool
                      █ Enabled: True
                      █ Found: False
                      █ THE MANAGER WAS NOT FOUND. PERHAPS IT IS NOT 
                      █ INSTALLED OR IT HAS BEEN MISCONFIGURED 
                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[21.08.2024 18:28:35] Command pwsh.exe was not found on the system
[21.08.2024 18:28:35] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                      █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                      █ Name: PowerShell7
                      █ Enabled: True
                      █ Found: False
                      █ THE MANAGER WAS NOT FOUND. PERHAPS IT IS NOT 
                      █ INSTALLED OR IT HAS BEEN MISCONFIGURED 
                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[21.08.2024 18:28:35] UniGetUI is up to date
[21.08.2024 18:28:35] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                      █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                      █ Name: Pip
                      █ Enabled: True
                      █ Found: True
                      █ Fancy exe name: pip
                      █ Executable path: C:\Users\Foo\AppData\Local\Microsoft\WindowsApps\python.exe
                      █ Call arguments:  -m pip
                      █ Version: 
                      █   
                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[21.08.2024 18:28:35] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                      █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                      █ Name: Winget
                      █ Enabled: True
                      █ Found: True
                      █ Fancy exe name: winget.exe
                      █ Executable path: C:\Users\Foo\AppData\Local\Microsoft\WindowsApps\winget.exe
                      █ Call arguments: 
                      █ Version: 
                      █   System WinGet CLI Version: v1.8.1911
                      █   Microsoft.WinGet.Client PSModule version: ""
                      █   Using Native WinGet helper (COM Api)
                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[21.08.2024 18:28:36] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                      █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                      █ Name: Scoop
                      █ Enabled: True
                      █ Found: True
                      █ Fancy exe name: scoop
                      █ Executable path: C:\WINDOWS\system32\windowspowershell\v1.0\powershell.exe
                      █ Call arguments:  -NoProfile -ExecutionPolicy Bypass -Command scoop
                      █ Version: 
                      █   Current Scoop version:
                      █   859d1db5 chore(release): Bump to version 0.5.2 (#6080)
                      █   
                      █   'main' bucket:
                      █   8d2845fee4 qsv: Update to version 0.132.0
                      █   
                      █   'extras' bucket:
                      █   808fe6e7b virtualhere-client: Update to version 5.7.5
                      █   
                      █   'versions' bucket:
                      █   57382a875 kicad-nightly: Update to version 8.99.0.2108.g0d990932c7
                      █   
                      █   'java' bucket:
                      █   84b7bb31 temurin11-nightly-jre: Update to version 11.0.25-3.0.ea
                      █   
                      █   'bissli' bucket:
                      █   c01b5ca added irontcl
                      █   
                      █   'gaming' bucket:
                      █   d3c6d04 nvidia-display-driver: Update to version 560.94
                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[21.08.2024 18:28:36] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                      █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                      █ Name: Chocolatey
                      █ Enabled: True
                      █ Found: True
                      █ Fancy exe name: choco.exe
                      █ Executable path: C:\Users\Foo\AppData\Local\UniGetUI\Chocolatey\choco.exe
                      █ Call arguments: 
                      █ Version: 
                      █   2.2.2
                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[21.08.2024 18:28:36] ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                      █▀▀▀▀▀▀▀▀▀▀▀▀▀ MANAGER LOADED ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
                      █ Name: PowerShell
                      █ Enabled: True
                      █ Found: True
                      █ Fancy exe name: powershell.exe
                      █ Executable path: C:\WINDOWS\system32\windowspowershell\v1.0\powershell.exe
                      █ Call arguments:  -NoProfile -Command
                      █ Version: 
                      █   Name                           Value                                                                                   
                      █   ----                           -----                                                                                   
                      █   PSVersion                      5.1.26100.1301                                                                          
                      █   PSEdition                      Desktop                                                                                 
                      █   PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                 
                      █   BuildVersion                   10.0.26100.1301                                                                         
                      █   CLRVersion                     4.0.30319.42000                                                                         
                      █   WSManStackVersion              3.0                                                                                     
                      █   PSRemotingProtocolVersion      2.3                                                                                     
                      █   SerializationVersion           1.1.0.1
                      ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
[21.08.2024 18:28:37] Microsoft.Powershell.Client detection has been forcefully skipped as the module is not required on the current context
[21.08.2024 18:28:37] Dependency WinGet PowerShell Module for manager Winget is present
[21.08.2024 18:28:37] Found 0 available updates from Pip
[21.08.2024 18:28:37] Found 0 installed packages from Pip
[21.08.2024 18:28:37] Dependency Scoop-Search for manager Scoop is present
[21.08.2024 18:28:37] Dependency Git for manager Scoop is present
[21.08.2024 18:28:37] LoadComponentsAsync finished executing. All managers loaded. Proceeding to interface.
[21.08.2024 18:28:38] Found 29 installed packages from Scoop
[21.08.2024 18:28:38] Found 65 installed packages from Winget
[21.08.2024 18:28:38] Found 1 installed packages from Chocolatey
[21.08.2024 18:28:38] Found 0 available updates from Winget
[21.08.2024 18:28:38] Found 0 installed packages from PowerShell
[21.08.2024 18:28:38] Found 0 available updates from Chocolatey
[21.08.2024 18:28:39] Found 0 available updates from PowerShell
[21.08.2024 18:28:43] Found 29 installed packages from Scoop
[21.08.2024 18:28:47] Found 0 available updates from Scoop
```

### Package Managers Logs

```text
▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
                           
Starting remove source operation for source name=gingwith Manager name=Scoop
Process Executable     : C:\WINDOWS\system32\windowspowershell\v1.0\powershell.exe
Process Call Arguments :  -NoProfile -ExecutionPolicy Bypass -Command scoop bucket rm ging
Working Directory      : C:\Users\Foo
Process Start Time     : 21.08.2024 18:42:55
    | ERROR 'ging' bucket not found.
Process Exit Code      : 1
Process End Time       : 21.08.2024 18:42:55
```

### Screenshots and videos

![2024-08-21_18-34](https://github.com/user-attachments/assets/5bfa7fc7-f3f7-4bc1-b9d1-3add6a0be000)

**Comment by marticliment** [comment]
Huh, this is weird.

Please try to add the source manually from cmd and share the output

**Comment by Daviteusz** [comment]
I tested adding this repo via powershell, cmd and via UniGetUI, in all cases UniGetUI displays `gaming` as `ging`.

In the attached logs, UniGetUI correctly recognizes the repo as `gaming` (or this is output from scoop command).

I renamed the repo into `test`, but UniGetUI still displays wrong repo link: `...\Scoop-Ging.git` instead `...\Scoop-Gaming.git`.

<img src="https://github.com/user-attachments/assets/5bfcf344-5bcf-4f4f-837e-adb89de334fd" width=30%>
